FeedAPIChallenge/Feed API/FeedItemMapper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Root.swift
+//  FeedAPIChallenge
+//
+//  Created by Gordon Feng on 13/6/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+public final class FeedItemMapper {
+	private struct Root: Codable {
```

Comment:
`Codable` is a type alias for the `Encodable` and `Decodable` protocols. In this case, we only need to decode `Root` from an external representation so `Codable` can be replaced with `Decodable`:
```suggestion
	private struct Root: Decodable {
```